src/operation.js

```diff
@@ -235,7 +235,7 @@ export class Operation {
         result.destination = encodeMuxedAccountToAddress(attrs);
         break;
       }
-      case 'manageDatum': {
+      case 'manageData': {
```

Comment:
These changes seem like they'll break old code; is that change documented anywhere? Is there a way to make this backwards-compatible?

Reply:
@morleyzhi  People who rely on stellar-base and the helper we expose, won't have any problem since the change is transparent for them.

However, if someone is using the autogenerated code for the XDR library and it's relying on this, it's very likely their code will break. I'll update the changelog to reflect this change.